### DISCUSSION TITLE
Release note and artefacts URLs customisation

### DISCUSSION THREAD

**Discussion by nboukeffa** [goreleaser/goreleaser#1618] [Q&A] [answered]
Hi,

I'm using GitLab Enterprise and Artifactory. The build, release and publish are working great, the only problem I have is that the URLs of the generated artefacts point to Gitlab instead of Artifactory inside the Release Note created on Gitlab.

So I was wondering if there is a trick to customise the URLs the artefacts present in a Release points to.

Thanks for your help.

**Comment by caarlos0** [comment, marked as answer]
Hi,

no, that list is the artifacts uploaded to gitlab itself, is not rendered by goreleaser.

what you might do is using `--release-footer` to point to the other artifacts, although getting the URL right might be a bit tricky.

check `goreleaser release --help` for more options, and also the [templating variables available](https://goreleaser.com/customization/templates/)

**Comment by nboukeffa** [comment]
Thanks for your answer, 

looking at the doc I don't see how I can loop over all the artefacts using the available variables.

I could hardcode the link for each platform in the template then I will only need the release version to build the link, but as soon I'll add a new platform I'll have to modify the template.

Maybe you have a better solution ?

Also is it possible to not generate artefact on Gitlab. Since I'm generating them already on Artifactory there is not sens to store it twice. I tried to set `release.disable` to `false`, but in this case no release / release note are generated.

Thanks for your help !

**Reply by caarlos0** [reply]
> I could hardcode the link for each platform in the template then I will only need the release version to build the link, but as soon I'll add a new platform I'll have to modify the template.
>
> Maybe you have a better solution ?

yeah, that's it unfortunately.

maybe we could add them to the release notes as we do with docker images... feel free to open an issue about that (although I can't promise anything)

> Also is it possible to not generate artefact on Gitlab. Since I'm generating them already on Artifactory there is not sens to store it twice. I tried to set `release.disable` to `false`, but in this case no release / release note are generated.

right now its not possible, no :/

**Reply by nboukeffa** [reply]
Thanks for your answer, I will open a feature request and maybe work on it if I find time.

And thank you again for your work on this tool ❤️
